compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/interop/ExternalMethodGen.java

```diff
@@ -200,13 +200,13 @@ public static OldStyleExternalFunctionWrapper createOldStyleExternalFunctionWrap
     }
 
     public static String getExternMethodDesc(List<BType> paramTypes, BType retType) {
-        return INITIAL_METHOD_DESC + JvmCodeGenUtil.populateMethodDesc(paramTypes) +
+        return INITIAL_METHOD_DESC + JvmCodeGenUtil.getMethodDescParams(paramTypes) +
                 generateExternReturnType(retType);
     }
 
     public static String getExternMethodDesc(List<BType> paramTypes, BType retType, BType attachedType) {
         return INITIAL_METHOD_DESC + JvmCodeGenUtil.getArgTypeSignature(attachedType) +
-                JvmCodeGenUtil.populateMethodDesc(paramTypes) + generateExternReturnType(retType);
+                JvmCodeGenUtil.getMethodDescParams(paramTypes) + generateExternReturnType(retType);
```

Comment:
Line not covered. may be unused method?

Reply:
This method is used in `createOldStyleExternalFunctionWrapper` method which is used in `createExternalFunctionWrapper` method. By the existing tests there the `else` part of an `if` condition does not get triggered. https://app.codecov.io/gh/ballerina-platform/ballerina-lang/blob/master/compiler%2Fballerina-lang%2Fsrc%2Fmain%2Fjava%2Forg%2Fwso2%2Fballerinalang%2Fcompiler%2Fbir%2Fcodegen%2Finterop%2FExternalMethodGen.java#L238 Hence, the whole `createOldStyleExternalFunctionWrapper` method is shown as not covered.